lib/thor/actions/file_manipulation.rb

```diff
@@ -117,7 +117,13 @@ def template(source, *args, &block)
       context = config.delete(:context) || instance_eval("binding")
 
       create_file destination, nil, config do
-        content = CapturableERB.new(::File.binread(source), nil, "-", "@output_buffer").tap do |erb|
+        match = ERB.version.match(/(\d\.\d.\d)/)
```

Comment:
My previous commit did not have a necessary backslash for the second "." 
```ruby
match = ERB.version.match(/(\d\.\d\.\d)/)
```

Here the dot should be actual dot character not any one character.

Reply:
I couldn't write a more correctly regular expression because of lack of self review 💦 I fixed it.

Reply:
Thank you for the update. I believe we did our best.